mmv1/third_party/terraform/tests/resource_org_policy_custom_constraint_test.go.erb

```diff
@@ -0,0 +1,102 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccOrgPolicyCustomConstraint_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        getTestOrgFromEnv(t),
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckOrgPolicyCustomConstraintDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccOrgPolicyCustomConstraint_v1(context),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Unless we're really concerned about individual values, we can omit resourceattr checks. They were useful at a time when most resources couldn't be imported, which automatically verifies that the states match. https://github.com/hashicorp/terraform-provider-google/wiki/Developer-Best-Practices#test-checks discusses a bit (but is a somewhat stale piece of documentation)

Reply:
Oh very cool, I didn't realize the imports did that for us 👍